core/src/main/scala/zio/jdbc/ZResultSet.scala

```diff
@@ -27,6 +27,9 @@ import java.sql.ResultSet
  */
 final class ZResultSet(private[jdbc] val resultSet: ResultSet) {
   def access[A](f: ResultSet => A): ZIO[Any, Throwable, A] = ZIO.attemptBlocking(f(resultSet))
+  def close                                                = ZIO.attempt(resultSet.close()).ignoreLogged
```

Comment:
Question: Is is a valid pattern to use `.ignoreLogged` in potentially failing finalizes like when using `.close()`?

Reply:
Yes, I prefer it myself.